crates/cdk-mintd/src/setup.rs

```diff
@@ -216,12 +216,14 @@ impl LnBackendSetup for config::FakeWallet {
             min_fee_reserve: self.reserve_fee_min,
             percent_fee_reserve: self.fee_percent,
         };
+        // calculate
+        let delay_time = (self.min_delay_time + self.max_delay_time) / 2;
```

Comment:
We don't want to do the average it will always be the same so there wouldn't be a point to accepting the range we could just accept what the delay should be. We want to simulate the variance between the range. If we want a specific delay 0 for example set both the min and max as 0.

Reply:
so we want generate a variance between the two value

`rng.gen_range(self.min_delay_time..=self.max_delay_time)`